internal/pipeline/parser_test.go

```diff
@@ -185,24 +185,31 @@ func TestParserParsesNoSteps(t *testing.T) {
 }
 
 func TestParserParsesGroups(t *testing.T) {
+	envMap := env.FromSlice([]string{`ENV_VAR_FRIEND="friend"`})
+
 	input := strings.NewReader(`---
 steps:
   - group:
     steps:
```

Comment:
I think this would fail without a label - needs to either be in the group key, or separate with group: null
```suggestion
  - group:
    label: "Group label"
    steps:
```

Reply:
oh nevermind, this doesn't fail the upload. just verified locally